code/modules/urist/modules/GOAI/integrations/blocker_definitions/machinery.dm

```diff
@@ -2,6 +2,12 @@
 
 // MACHINERY
 
+/obj/machinery
+	blocker_gen_enabled = TRUE
+
```

Comment:
I've gone through these, I don't think we can afford to default to generating here. A whole lot of machinery should be non-blocking - wires, buttons and such.

Reply:
The default `GenerateGenericFullBlocker()` will only do it's thing (by default) if the `/obj/machinery` is dense. As it only gets called when Astar checks it and it's then cached to `blocker_gen_enabled = FALSE`, would this really be significant overhead in return for some definition shortcuts, as it'll only run once _if_ Astar pokes it? Idea was for things that shouldn't generate, we could just override it by default?
Happy to reverse this though if you think it will be!

Reply:
Ah, you're right. I think I need to work on some more docs for this, the complexity of this project is getting overwhelming.